apps/web/src/pages/InstructorLedTrainingPage/InstructorLedTrainingPage.tsx

```diff
@@ -295,14 +313,21 @@ export const Component = () => {
               {intl.formatMessage(
                 {
                   defaultMessage:
-                    "The training opportunities on this page are supported by the <link>IT Community Training and Development Fund</link> and are available only to employees represented by PIPSC in the IT group.",
-                  id: "KUuhwO",
+                    "The training opportunities on this page are supported by the <itTrainingFundLink>IT Community Training and Development Fund</itTrainingFundLink> and are available only to IT-classified employees who are covered by the <itCollectiveAgreementLink>IT collective agreement.</itCollectiveAgreementLink>",
+                  id: "AQybBB",
```

Comment:
Not really in scope for this but the use of IT is going to cause issues for screen readers. Some will pronounce it as "it" instead of "I T"